trustyai_tests/tests/explainers/kserve/test_kserve_explainers.py

```diff
@@ -0,0 +1,20 @@
+from trustyai_tests.tests.utils import (
+    verify_model_prediction,
+    verify_saliency_explanation
+)
+
+class TestExplainers:
+    def test_bank_churn_model_prediction(self, model_namespace, inference_service, service_hostname):
+        verify_model_prediction(
+            model_namespace, inference_service, service_hostname
+        )
+
+    def test_request_lime(self, model_namespace, inference_service, service_hostname):
+        verify_saliency_explanation(
+            model_namespace, inference_service, service_hostname
+        )
+
+    def test_request_shap(self, model_namespace, inference_service, service_hostname):
```

Comment:
This test and the previous one are the same, right? Shouldn't we pass a parameter to verify_saliency_explanation to differentiate if we are requesting LIME or SHAP?

Reply:
AFAIK there is no way to differentiate between the LIME and SHAP inference response. However, in the InferenceService spec, we specify which explainer we want to use

Reply:
@adolfo-ab @christinaexyou This is something we should actually add to the explainer (an "explanation type" field). I'll open an issue and link here, but I don't think this PR should wait for that, we can change it on a separate PR.

Reply:
https://github.com/trustyai-explainability/trustyai-kserve-explainer/issues/8